service/controller/resource/controlplanestatus/resource.go

```diff
@@ -0,0 +1,107 @@
+package controlplanestatus
+
+import (
+	"context"
+
+	infrastructurev1alpha2 "github.com/giantswarm/apiextensions/pkg/apis/infrastructure/v1alpha2"
+	"github.com/giantswarm/k8sclient/v3/pkg/k8sclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/controller/context/reconciliationcanceledcontext"
+
+	"github.com/giantswarm/cluster-operator/pkg/label"
+	"github.com/giantswarm/cluster-operator/service/controller/key"
+	"github.com/giantswarm/cluster-operator/service/internal/nodecount"
+)
+
+const (
+	Name = "controlplanestatus"
+)
```

Comment:
Just FYI with `operatorkit@v2.0.0` we will not need this anymore. I just need to have time to work on this. 🙈

Reply:
Oh cool 👍